includes/Checker/Runtime_Environment_Setup.php

```diff
@@ -100,4 +100,42 @@ public function cleanup() {
 			}
 		}
 	}
+
+	/**
+	 * Checks if the Runtime Environment can be created.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool Returns true if the runtime environment can be created, false if not.
+	 */
+	public function can_setup_runtime_environment() {
+		global $wp_filesystem;
+
+		require_once ABSPATH . '/wp-admin/includes/upgrade.php';
+
+		if ( ! $wp_filesystem && ! WP_Filesystem() ) {
+			return false;
+		}
```

Comment:
Instead of checking this can we check something below?

```
if ( $wp_filesystem && WP_Filesystem() ) {
        // Check if the object-cache.php file exists.
		if ( $wp_filesystem->exists( WP_CONTENT_DIR . '/object-cache.php' ) ) {
		.....
} else {
	return false;
}
```

Reply:
I don't like the above suggestion.

Reply:
Agreed, early returns are preferable over nesting which leads to heavily indented / harder-to-read code.

Reply:
Marking this as resolved